snap/snapcraft.yaml

```diff
@@ -277,26 +271,47 @@ parts:
       - libattr1-dev
       - libfdt-dev
       - curl
+      - libcapstone-dev
     build-snaps:
       - yq
     override-build: |
-      chmod +x ${SNAPCRAFT_STAGE}/qemu/scripts/configure-hypervisor.sh
-      # static build
-      echo "$(${SNAPCRAFT_STAGE}/qemu/scripts/configure-hypervisor.sh -s qemu) \
-        --disable-rbd
-        --prefix=/snap/${SNAPCRAFT_PROJECT_NAME}/current/usr \
-        --datadir=/snap/${SNAPCRAFT_PROJECT_NAME}/current/usr/share \
-        --libexecdir=/snap/${SNAPCRAFT_PROJECT_NAME}/current/usr/libexec/qemu" \
-        | xargs ./configure
+      # set GOPATH
```

Comment:
there variables are not required, because qemu can be cloned and built in the current directory